X/XPA/build_tarballs.jl

```diff
@@ -3,22 +3,21 @@
 using BinaryBuilder, Pkg
 
 name = "XPA"
-version = v"2.1.19"
+version = v"2.1.20"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/ericmandel/xpa.git", "c0452e139134d6d1677b5e6fda2ad5283c8ba4c7"),
+    GitSource("https://github.com/ericmandel/xpa.git", "923cc1bc7e761424b87049b1a20853eefe921388"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/xpa
```

Comment:
This should do the trick for FreeBSD
```suggestion
cd $WORKSPACE/srcdir/xpa
if [[ "${target}" == *-freebsd* ]]; then
    export CFLAGS=-fPIC
fi
```